Specs/blob/blob-storage-2017-07-29.json

```diff
@@ -3711,6 +3922,122 @@
         }
       ]
     },
+    "/{containerName}/{blob}?comp=page&clear": {
+      "put": {
+        "tags": [
+          "pageblob"
+        ],
+        "operationId": "PageBlob_ClearPages",
+        "description": "The Clear Pages operation clears a set of pages from a page blob",
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/Body"
+          },
+          {
+            "$ref": "#/parameters/ContentLength"
+          },
+          {
+            "$ref": "#/parameters/Timeout"
+          },
+          {
+            "$ref": "#/parameters/Range"
+          },
+          {
+            "$ref": "#/parameters/PageWriteClear"
+          },
+          {
+            "$ref": "#/parameters/LeaseId"
```

Comment:
Note to check that when the other PR gets merged that all of these are updated as well